lib/oli_web/controllers/page_delivery_controller.ex

```diff
@@ -287,7 +287,8 @@ defmodule OliWeb.PageDeliveryController do
             section.slug,
             context.page.slug,
             resource_attempt.attempt_guid
-          )
+          ),
+        screenIdleTimeOut: Application.fetch_env!(:oli, :screen_idle_timeout)
```

Comment:
This is the property that delivery is expecting to get from server. I am not sure if there already exists a key that can be passed to delivery. I added 'SCREEN_IDLE_TIMEOUT' in oli.env file that holds the screen time out value.